data/emulator.js

```diff
@@ -2573,188 +2578,188 @@ class EmulatorJS {
-                {"type":"button","text":"Select","id":"select","location":"center","left":-5,"fontSize":15,"block":true,"input_value":2},
-                {"type":"button","text":"L","id":"l","location":"left","left":3,"top":-90,"bold":true,"block":true,"input_value":10},
-                {"type":"button","text":"R","id":"r","location":"right","right":3,"top":-90,"bold":true,"block":true,"input_value":11}
-            ];
+                {"type":"button","text":this.localization("Start"),"id":"start","location":"center","left":60,"fontSize":15,"block":true,"input_value":3},
+                {"type":"button","text":this.localization("Select"),"id":"select","location":"center","left":-5,"fontSize":15,"block":true,"input_value":2},
+                {"type":"button","text":this.localization("L"),"id":"l","location":"left","left":3,"top":-90,"bold":true,"block":true,"input_value":10},
+                {"type":"button","text":this.localization("R"),"id":"r","location":"right","right":3,"top":-90,"bold":true,"block":true,"input_value":11}
+            ];            
             info.push(...speedControlButtons);
         } else if ("gb" === this.getControlScheme()) {
             info = [
-                {"type":"button","text":"A","id":"a","location":"right","left":81,"top":40,"bold":true,"input_value":8},
-                {"type":"button","text":"B","id":"b","location":"right","left":10,"top":70,"bold":true,"input_value":0},
+                {"type":"button","text":this.localization("A"),"id":"a","location":"right","left":81,"top":40,"bold":true,"input_value":8},
+                {"type":"button","text":this.localization("B"),"id":"b","location":"right","left":10,"top":70,"bold":true,"input_value":0},
                 {"type":"dpad","location":"left","left":"50%","top":"50%","joystickInput":false,"inputValues":[4,5,6,7]},
-                {"type":"button","text":"Start","id":"start","location":"center","left":60,"fontSize":15,"block":true,"input_value":3},
-                {"type":"button","text":"Select","id":"select","location":"center","left":-5,"fontSize":15,"block":true,"input_value":2}
+                {"type":"button","text":this.localization("Start"),"id":"start","location":"center","left":60,"fontSize":15,"block":true,"input_value":3},
+                {"type":"button","text":this.localization("Select"),"id":"select","location":"center","left":-5,"fontSize":15,"block":true,"input_value":2}
             ];
             info.push(...speedControlButtons);
         } else if ('nes' === this.getControlScheme()) {
             info = [
-                {"type":"button","text":"B","id":"b","location":"right","right":75,"top":70,"bold":true,"input_value":0},
-                {"type":"button","text":"A","id":"a","location":"right","right":5,"top":70,"bold":true,"input_value":8},
+                {"type":"button","text":this.localization("B"),"id":"b","location":"right","right":75,"top":70,"bold":true,"input_value":0},
```

Comment:
This is a little messy, can we remove any calls to localization within these if statements and just cycle through the info object and set these afterwards

Reply:
Yes lol that's a good idea, don't know why I did not think of that.

Reply:
lol I forgot this